site/en/blog/optimizing-lcp-using-signed-exchanges/index.md

```diff
@@ -0,0 +1,459 @@
+
+{% Img src="image/rULxC7pPw3PFS4o9xr7v8isFmCv1/MNaYo8ZjOxto57txJvwf.png",
+       alt="Google Analytics Events report with SXG segment, showing 12.5% Unique Events",
+       width="363", height="111" %}
+
+Finally, go to the [Web Vitals Report](https://web-vitals-report.web.app/), select "Choose segments", and select "SXG counterfactual" and "SXG".
+
+{% Img src="image/rULxC7pPw3PFS4o9xr7v8isFmCv1/tIsSMUofs9lMv3C0ncwe.png",
+       alt="Web Vitals Report with selections for SXG counterfactual and SXG",
+       width="657", height="164" %}
+
+Click "Submit", and you should see LCP distributions for the two segments. This should fill in the Y for "improving their LCP by Y milliseconds at the 75th percentile":
+
+{% Img src="image/rULxC7pPw3PFS4o9xr7v8isFmCv1/LTiuLf05jVOH2xe9RMOg.png",
+       alt="Web Vitals Report showing LCP distributions for SXG counterfactual and SXG",
+       width="800", height="363" %}
+
+#### Caveats
+
+Once you've applied all of the above filters, SXG counterfactual page loads should consist of things like these:
+
+  - cache misses — If the Google SXG Cache doesn't have a fresh copy of the SXG for a given URL, it will redirect to the original URL at your site.
+  - other result types — As of writing, Google Search only supports SXG for standard web results (aka "10 blue links") and a few other types. Others, like Top Stories Carousel, will link to the original URL at your site.
+  - ineligible URLs — If some pages on your site are not eligible for SXG (e.g. because they are not cacheable), they could appear in this set.
+
+There may be remaining bias between the SXG page loads and the above set of non-SXG page loads, but I believe it will be smaller in magnitude than the above biases. For instance, perhaps your non-cacheable pages are slower or faster than your cacheable pages. If you suspect this could be an issue, consider looking at the data limited to a specific SXG-eligible URL to see if its results match the overall study.
+
+If your site has some AMP pages, they probably won't see performance improvements from enabling SXG, as they can already be prefetched from Google Search. Consider adding a filter to exclude such pages, to further "zoom in" on the relevant changes.
+
+Lastly, even addressing all selection biases, there is risk that survivorship bias makes LCP improvements look like degradations in RUM statistics. [This article](https://simonhearne.com/2022/survorship-bias-in-webperf/) does a great job of explaining that risk, and suggests looking at some form of abandonment metric to detect whether this is happening.
+
+### Before/after study
+
+To corroborate results from the contemporary study, it may be useful to do a comparison of LCP before and after enabling SXG. Don't limit to SXG page views, to eliminate the potential biases noted above. Instead, look at SXG-eligible results — the above segment definitions but without the **isSXG** constraint.
```

Comment:
```suggestion
To corroborate results from the contemporary study, it may be useful to do a comparison of LCP before and after enabling SXG. Don't limit to SXG page views, to eliminate the potential biases noted above. Instead, look at SXG-eligible results—the above segment definitions but without the `isSXG` constraint.
```